conftest.py

```diff
@@ -213,6 +213,7 @@ def database_table_fixture(request):
     database_name_to_conn_id = {
         Database.SQLITE: "sqlite_default",
         Database.BIGQUERY: "google_cloud_default",
+        Database.POSTGRES: "postgres_conn",
```

Comment:
The default is `postgres_default`

https://github.com/apache/airflow/blob/e36868ba1b8a26f838851d1d09cca9db64cbcdc8/airflow/providers/postgres/hooks/postgres.py#L62

```suggestion
        Database.POSTGRES: "postgres_default",
```